samples/cpp/tutorial_code/ImgProc/Pyramids.cpp

```diff
@@ -39,15 +39,14 @@ int main( void )
   dst = tmp;
 
   //![create_window]
-  namedWindow( window_name, WINDOW_AUTOSIZE );
   imshow( window_name, dst );
   //![create_window]
 
   //![infinite_loop]
   for(;;)
   {
     int c;
-    c = waitKey(10);
+    c = waitKey(0);
 
     if( (char)c == 27 )
```

Comment:
Also this line then becomes `if ( c == 27 )`, the explicit char conversion can lead to precision and data loss.